src/day5/main2.java

```diff
@@ -0,0 +1,209 @@
+package day5;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class main2 {
+
+    public static void printInformation(List dataBase) {
```

Comment:
Good. You can read about and use "toString()" method to make this piece of code more optimal.